js/siteorigin-panels/view/widgets/text-widget.js

```diff
@@ -0,0 +1,17 @@
+var $ = jQuery;
+
+var textWidget = {
+	addWidget: function( idBase, widgetContainer, widgetId ) {
+		var component = wp.textWidgets;
+
+		var widgetControl = new component.TextWidgetControl({
```

Comment:
We need to check for the existence of `TextWidgetControl`, from my comment here - https://github.com/siteorigin/siteorigin-panels/pull/425#issuecomment-304324650